scripts/prerm

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+if [ -L ${CMAKE_INSTALL_FULL_LIBDIR}/python3/dist-packages/TestHarness ]; then
+    rmdir -p ${CMAKE_INSTALL_FULL_LIBDIR}/python3/dist-packages/TestHarness
+fi
+
+if [ -L ${CMAKE_INSTALL_FULL_DATAROOTDIR}/leap_testing/bin ]; then
+    rmdir -p ${CMAKE_INSTALL_FULL_DATAROOTDIR}/leap_testing/bin
+fi
```

Comment:
I don't think `rmdir -p` is the correct command to remove a symbolic link. I think we can just `rm -f` these symlinks, existing or not. Plus, simpler is better, righ?

```suggestion
rm -f ${CMAKE_INSTALL_FULL_LIBDIR}/python3/dist-packages/TestHarness
rm -f ${CMAKE_INSTALL_FULL_DATAROOTDIR}/leap_testing/bin
```

Reply:
Addressed: [08b1eb5](https://github.com/AntelopeIO/leap/pull/818/commits/08b1eb525242be8746f68888b66cca9aae84be51)